docs/development/jump-menu.md

```diff
@@ -17,33 +17,50 @@ Add-ons can easily add custom items for the Jump Menu.
 
 NOTE: Sample File: https://gist.github.com/obfuscode/77d0135970c9ce8bc84139c127507bb1
 
-1. Create a `jump.[addon_name].php` file in your add-on folder
+Create a `jump.[addon_name].php` file in your add-on folder
 
-2. Your jump file MUST have the following:
+Your jump file MUST have the following:
 
         use ExpressionEngine\Service\JumpMenu\AbstractJumpMenu;
     
         class [AddonName]_jump extends AbstractJumpMenu
         {
 
-        protected static $items = [];
+        protected static $items = array(
+          'commandArrayTitle' => array(
+            'icon' => 'fa-file',
+            'command' => 'few lowercase words to be fuzzy-matched in jump menu',
+            'command_title' => 'Displayed <b>command title upon match from above</b>',
+            'dynamic' => false,
+            'requires_keyword' => false,
+            'target' => 'See Below. Behavior changes based on dynamic element above'
+        ));
 
-3. Add commands for your custom items.   
-Item commands are comprised of the following:
+To Add Jump Menu commands to your add-on, you simply add array elements to the $items array in the example above
 
-- `[commandTitle]` _(string)_ Unique command title used as key in global jumps array. Will be prefixed with `[addonName]:` so a command title in Assets of `editS3Source` will be `Assets:editS3Source`
+The array of a Jump Menu command is comprised of the following keys:
+
+- `commandArrayTitle` _(string)_ Unique command title used as key in global jumps array. Will be prefixed with the add-ons name so a command title in the add-on Assets of `editS3Source` will be `Assets:editS3Source`
```

Comment:
I think this should be `commandTitle` and not `commandArrayTitle`?

Also I feel like the text needs some proofreading - tagging @matthewjohns0n

Reply:
Theres a second command_title lower as a secondary array element, that has to be kept... Changed it to array title to make it unique, so people would know which one is being referenced.